grpc/src/main/java/com/linecorp/armeria/server/grpc/HttpJsonTranscodingService.java

```diff
@@ -348,7 +348,7 @@ private static Map<String, Field> buildFields(Descriptor desc,
                                 .findFirst().orElse(null);
                     if (typeDesc == null) {
                         // From the proto file.
-                        typeDesc = findTypeDescriptor(desc.getFile(), field);
```

Comment:
So probably we can replace the usage of `findTypeDescriptor` to `field.getMessageType()` in this class?
Could you check it please?

Reply:
It seems that `findTypeDescriptor` is no longer needed because we don't directly find the descriptor from files anymore. I removed it in [03aea23](https://github.com/line/armeria/pull/4641/commits/03aea2307fac79a1183acce6abcbe75db6ad2a94)